RecoJets/JetProducers/python/ECF_cfi.py

```diff
@@ -2,6 +2,90 @@
 
```

Comment:
cfi files shouldn't contain the definition of multiple configs: please rename it *_cff.py

Moreover, the primary cfi file with all defaults is now automatically created by fillDescription(): therefore, in the new cff you can clone that default config and only modify the parameter(s) who need so, thus avoiding a lot of copy/paste